### ISSUE TITLE
Integrate Bloom's Copyright & License dialog

### ISSUE THREAD

**Issue by papeh** [sillsdev/libpalaso#1273]
The Bloom team has implemented an easier-to-understand CC license picker. Could this be integrated into the Palaso ClearShare library?

The Bloom team has also designed (but not implemented) an entire new image picker, which we will want to integrate whenever it gets implemented (currently not a priority for the Bloom team).

Palaso ClearShare's current dialog:
![image](https://github.com/sillsdev/libpalaso/assets/4933670/9cee59b3-84ef-4e9f-809f-7eb1db3f42a4)

Bloom's current dialogs:
![image](https://github.com/sillsdev/libpalaso/assets/4933670/cea850a0-9aa3-4ffb-9d7a-2013b6adf10f)

Bloom's proposed dialog:
https://www.figma.com/file/V5wQEtjPkOuLNY2Yu6UoUW4Z/Bloom-Edit-Tab?type=design&node-id=3326-7453&mode=design

**Comment by andrew-polk** [comment]
Note, the old forms are WinForms.
The new ones run in a browser (components are built using React).

**Comment by hatton** [comment]
With our browser-based components, we would, of course, like to continue the tradition of SIL LSDev co-ownership/maintenance that we have with libpalaso. However this time around, we think a collection of independent libraries is a better way forward. In rare cases ([config-r](https://github.com/sillsdev/config-r), [comicaljs](https://github.com/BloomBooks/comical-js)), we are building components as independent NPM libraries from the start. In other cases, where it isn't clear if anyone else will use a component, we'll wait for another project to actually be ready to use the component before going to the work of spinning it off.

**Comment by imnasnainaec** [comment]
Since Bloom's dialog is a React component, even if there's demand for it to become an independent NPM library, that won't be in libpalaso.

**Comment by hatton** [comment]
Yeah nowadays it is react. But at the time of this issue, the Bloom copyright and license dialog might still have been winforms. So if you really wanted, that code is available and could be used. But yeah probably better to use the new web-based on in a webview2 control if you can.

**Comment by tombogle** [comment]
@papeh , Was your thought perhaps to reimplement a similar design in WinForms, or possibly to use the React component inside a WinForms dialog with a WebView2 control?
